dask_cuda/explicit_comms/dataframe/shuffle.py

```diff
@@ -18,7 +18,7 @@
 from dask.base import tokenize
 from dask.dataframe import DataFrame, Series
 from dask.dataframe.core import _concat as dd_concat
-from dask.dataframe.shuffle import group_split_dispatch, hash_object_dispatch
+from dask.dataframe.dispatch import group_split_dispatch, hash_object_dispatch
```

Comment:
All dispatch functions have been centralized in the `dispatch` module for a long time now. Many changes in this PR are just using that preferred module.